langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -128,6 +186,11 @@ public isolated function reduce(map<Type> m, @isolatedParam function(Type1 accum
 
 # Removes a member of a map.
 #
+# ```ballerina
+# map<int> marks = {"Carl": 85, "Bob": 50, "Max": 60};
+# _ = marks.remove("Carl");
```

Comment:
Do we use `_` in our langlip apidocs?. I think we need to add a variable for this.

Reply:
Yeah, let's not ignore in examples.